### DISCUSSION TITLE
Duda JS - Imprimir en HTML

### DISCUSSION THREAD

**Discussion by OmarVenturaP** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#442] [General]
Quisiera de su apoyo para sacarme de una duda.

¿Sabrán que función o valor puede haber para imprimir lo que ya imprimí en consola?

Ya me aparecen en consola los valores que necesito, pero están dentro de una función. Pero al intentar llamar a la función me marca error de sintaxis.  Y dice que el valor (el mismo que imprimí en consola) no está declarado.

(Menciono que imprimí dentro de una función) no se si eso tendrá algo que ver.

**Comment by Drako9159** [comment]
primero llamas a tu ID en html y lo declaras, algo así `pokemon = document.getElementById("Id");`  luego para enviar texto lo haces como `pokemon.textContent = "El texto que quiero enviar"`

**Reply by OmarVenturaP** [reply]
Ok. Te agradezco el apoyo, solo una dudita mas:

En el "texto que quiero enviar" supongamos que sea una variable que aparece dentro de un "then" anterior. Se podrá? Has de cuenta en el código del MC. Te marco si de declare el let que comento que me imprime en consola.

![Screenshot_20220317-122708~2.png](https://user-images.githubusercontent.com/99059639/158871675-23cdc403-ef3a-4058-b128-c07b8eabf1e3.png)
